app/controllers/api_applications_controller.rb

```diff
@@ -36,25 +32,19 @@ def create
   def update
     @api_application = current_user.api_applications.find(params[:id])
 
-    respond_to do |format|
-      if @api_application.update_attributes(params[:api_application])
-        format.html do
-          redirect_to @api_application,
-                      notice: 'Application was successfully updated.'
-        end
-      else
-        format.html { render :edit }
-      end
+    if @api_application.update_attributes(params[:api_application])
+      redirect_to edit_api_application_path(@api_application),
+                  notice: 'Application was successfully updated.'
+    else
+      render :edit
     end
   end
 
   # DELETE /api_applications/1
   def destroy
-    @api_application = current_user.api_applications.find(params[:id])
-    @api_application.destroy
-
-    respond_to do |format|
-      format.html { redirect_to api_applications_url }
-    end
+    api_application = current_user.api_applications.find(params[:id])
+    api_application.destroy
+    redirect_to api_applications_path,
+                notice: 'Application was successfully deleted.'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.